collector/shards.go

```diff
@@ -156,20 +159,26 @@ func (s *Shards) Collect(ch chan<- prometheus.Metric) {
 	nodeShards := make(map[string]float64)
 
 	for _, shard := range sr {
-		if val, ok := nodeShards[shard.Node]; ok {
-			nodeShards[shard.Node] = val + 1
-		} else {
-			nodeShards[shard.Node] = 1
+		if shard.State == "STARTED" {
```

Comment:
This seems to change the behavior of the metric. What is the intention here?

Reply:
Ideally, this metrics should be able to tell how many shards each node is handling. The api which this exporter hit is `_cat/shards` which also lists down the relocating shards.

![image](https://user-images.githubusercontent.com/15676216/199859650-fbd3ee6a-a3f1-4b21-8ede-e99e94a7273c.png)

Those entries are disruptive and according to existing source code, `node` label gets that whole big string with an arrow and ip address which doesn't make sense.

So, It's better to only capture already relocated shards aka (STARTED)

Reply:
In the JSON, I see it's showing `UNASSIGNED` shards also. We only should be capturing `STARTED` shards

<img width="956" alt="image" src="https://github.com/prometheus-community/elasticsearch_exporter/assets/15676216/18564ba5-5343-457d-ab9d-c5136059bf95">

Reply:
if shards are RELOCATING. this is how it appears:
<img width="1470" alt="image" src="https://github.com/prometheus-community/elasticsearch_exporter/assets/15676216/6d0940ad-67c4-45af-82d2-223593a1b4d0">

Reply:
This metric only intents to tell how many shards each node currently have. Therefore, we must exclude `RELOCATING` & `UNASSIGNED`